lib/elixir/lib/module.ex

```diff
@@ -1727,6 +1736,23 @@ defmodule Module do
     :ets.insert(set, {key, value, line})
   end
 
+  # If any of the doc attributes are called with a keyword list that
+  # will become documentation metadata. Multiple calls will be merged
+  # into the same map overriding duplicate keys.
+  defp put_attribute(_module, key, {_, metadata}, line, set, _bag)
```

Comment:
Let's move this clause up so we don't need to change the guards of the clause before.

Reply:
Good catch, will do.